CONTRIBUTING.adoc

```diff
@@ -0,0 +1,184 @@
+
+The https://github.com/kubernetes/test-infra/tree/master/prow/external-plugins/cherrypicker[`/cherrypick` tool] can be use to automatically pull back changes from `master` to previous releases if necessary.
+
+=== Tools and Style
+
+Knative documentation follows the https://developers.google.com/style/[Google Developer Documentation Style Guide]. Use this as a reference for writing style questions. use it as a reference for writing style questions.
+
+Knative uses several sets of tools to manage pull requests (PR)s and issues in a more fine-grained way than GitHub permissions allow. In particular, you'll regularly interact with https://github.com/kubernetes/test-infra/tree/master/prow[Prow] to categorize and manage issues and PRs. Prow allows control of specific GitHub functionality without granting full "write" access to the repo (which would allow rewriting history and other dangerous operations). You'll most often use the following commands, but Prow will also chime in on most bugs and PRs with a link to all the known commands:
+
+* `/assign @user1 @user2` to assign an issue or PR to specific people for review or approval.
+* `/lgtm` and `/approve` to approve a PR. Note that _anyone_ may `/lgtm` a PR, but only someone listed in an `OWNERS` file may `/approve` the PR. A PR needs both an approval and an LGTM -- the `/lgtm` review is a good opportunity for non-approvers to practice and develop reviewing skills. `/lgtm` is removed when a PR is updated, but `/approve` is sticky -- once applied, anyone can supply an `/lgtm`.
+* Both Prow (legacy) and GitHub actions (preferred) can run tests on PRs; once all tests are passing and a PR has the `lgtm` and `approved` labels, Prow will submit the PR automatically.
+* You can also use Prow to manage labels on PRs with `/kind ...`, `/good-first-issue`, or `/area ...`
+* See <<branches>> for details on the `/cherrypick` command.
+
+[NOTE]
+====
+As an experimental process, tag tech writer rewritten documents with `reviewed: $DATE` (fill in the date of starting the PR). This allows finding documents which have not been reviewed with the `find` command (Mac and Linux):
+[source,shell]
+----
+find docs -type f -exec /bin/bash -c 'head -1 {} | grep ^--- >/dev/null && grep -L reviewed: {}' \;
+----
+(This currently finds all documents, but as docs are reviewed, will become more useful.)
+In the future, we may also be able to use this metadata to provide freshness information on the website.
```

Comment:
Erm, may want to leave this in the Docs Taskforce that's forming, feels like shoehorn of a docs process into this doc which hasn't been seen by any tech writers. 

CC: @RichardJJG @snneji

Reply:
SGTM, removing.

Reply:
But, you also have my proposal for how to manage / track review in the Git history here! 😉

Reply:
bahahha ack

Reply:
thoughts on this @RichieEscarez?

Reply:
Building on this. If we have the other WG TLs own the triaging process of all Issues and PRs, they could be the filter for the limited writing resources that we have today and therefore "tagging" a PR or Issue when it is indeed ready (ie. completed a SME tech review) would/might streamline the process (reduce noice and allow writers to focus solely on actionable items)